modules/administration-guide/pages/installing-che-on-red-hat-openshift-local.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+:navtitle: Installing {prod-short} on Red Hat OpenShift Local
+:keywords: overview, installing-che-on-red-hat-open-shift-local
+:page-aliases: installation-guide:installing-che-on-red-hat-open-shift-local, overview:installing-che-on-red-hat-open-shift-local
```

Comment:
* Always keep the old aliases, as they ensure redirection from the older publication location
* Always add an alias when you rename a page

```suggestion
:page-aliases: installation-guide:installing-che-on-codeready-containers, overview:installing-che-on-codeready-containers, 
installation-guide:installing-che-on-codeready-containers, overview:installing-che-on-codeready-containers
```